cfme/containers/image_registry.py

```diff
@@ -26,12 +27,29 @@ def __init__(self, host, provider):
         self.host = host
         self.provider = provider
 
+    def _on_detail_page(self):
+        return sel.is_displayed(
+            '//div//h1[contains(., "{} (Summary)")]'.format(self.host))
```

Comment:
Now you can use `summary_title()` from `cfme.web_ui` to read the title and compare it to a string.

You can also use the `breadcrumbs()` function from the same module to check the breadcrumbs to verify you are on that page.